pkg/testcase/clusterreset.go

```diff
@@ -0,0 +1,98 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestClusterReset() {
+	cluster := factory.ClusterConfig(GinkgoT())
+
+	killall(cluster)
+	shared.LogLevel("INFO", "%s-service killed", cluster.Config.Product)
+
+	stopServer(cluster)
+	shared.LogLevel("INFO", "%s-service stopped", cluster.Config.Product)
+
+	productLocationCmd := fmt.Sprintf("which %s", cluster.Config.Product)
```

Comment:
Actually can we change this command to be:
```suggestion
	productLocationCmd := fmt.Sprintf("sudo find / -type f -executable -name %s", cluster.Config.Product)
```

I think this will be better, because right now in SLES it appears to fail with:
```
 command: which rke2 failed on run ssh: a.bb.cc.ddd with error: Process exited with status 1
       , stderr: which: no rke2 in (/usr/sbin:/usr/bin:/sbin:/bin:/var/lib/rancher/rke2/bin:/opt/rke2/bin)
```